package/wpe/wpeframework-plugins/wpeframework-plugins.mk

```diff
@@ -4,8 +4,8 @@
 #
 ################################################################################
 
-WPEFRAMEWORK_PLUGINS_VERSION = 8b70bd116e65f8824f060e0e26a4b1cfe65e27cd
-WPEFRAMEWORK_PLUGINS_SITE = $(call github,WebPlatformForEmbedded,WPEFrameworkPlugins,$(WPEFRAMEWORK_PLUGINS_VERSION))
+WPEFRAMEWORK_PLUGINS_VERSION = 5d0769cea06138762ac8ecc102b5b8e5c9b3a863
+WPEFRAMEWORK_PLUGINS_SITE = $(call github,s-rapolu,WPEFrameworkPlugins,$(WPEFRAMEWORK_PLUGINS_VERSION))
```

Comment:
We cant point to your fork, this needs to be a commit on the main repository (master) that has the required changes to support the ssh server plugin.

Reply:
Sorry, corrected the mistake and now pointing it to master.